test/expected/tablespace.out

```diff
@@ -245,9 +245,39 @@ SELECT * FROM show_tablespaces('tspace_2dim');
 ------------------
 (0 rows)
 
---cleanup
-DROP TABLE tspace_1dim CASCADE;
-DROP TABLE tspace_2dim CASCADE;
+--cleanup - make sure tablespace metadata is removed
```

Comment:
Can we test the case that there are chunks using those tablespaces? Make sure all the cascade stuff works?

Reply:
Not sure which cascade test you are thinkingof. The only case for deleting tablespaces is during a DROP on the hypertable. In that case, all tablespaces and chunks should be removed.

Do you mean what happens when you do a DROP tablespae, although chunks are still using them?

Reply:
yes that's what I mean

Reply:
That is not possible. From PG docs: `The tablespace must be empty of all database objects before it can be dropped.`